### ISSUE TITLE
[Android] Still getting OpenAL Error for compressed streamed audio

### ISSUE THREAD

**Issue by shichen85** [YoYoGames/GameMaker-Bugs#7562]
### Description
We are still observing that in Android exports for compressed streamed audio with the following error messages:

```
m_pJavaVM was null for OGG thread
Error (thread index 0): 
Failed to open assets/mus_main.ogg
Error: Failed to read vorbis info for assets/grc_snd_wave.ogg
ov_open_callbacks returned -132 -> Bitstream/page/packet is not Vorbis data.
ov_read returned -3 -> There was an interruption in the data (one of: garbage between pages, loss of sync followed by recapture, or a corrupt page)
 OpenAL Error: 40963 (Invalid value)
     After: Detaching source buffer
     In file: /home/runner/work/GameMaker/GameMaker/GameMaker/Runner/GMS2-Runner-Main/Build/android/../../VC_Runner/Android/jni/../jni/yyal/../../../YYAL/src/Audio_Main.cpp
     In function: Audio_SetNoiseInactive
     On line: 1182
```
In this project, the runner seems to be able to continue to function despite the error. However, in our bigger project like Crashlands 2, the runner will freeze.

### Steps To Reproduce
1. Run the project for Android export VM in the `dev` config
2. Observe the error messages in the output

### Which version of GameMaker are you reporting this issue for?
IDE v2024.800.0.618 Runtime v2024.800.0.642

### Which operating system(s) are you seeing the problem on?
Windows 10.0.22000.0

### Which platform(s) are you seeing the problem on?
Android

[d2daed21-63e6-4c13-a00d-06162ca3ad68](https://api.gamemaker.io/api/github/downloads/d2daed21-63e6-4c13-a00d-06162ca3ad68)

**Comment by shichen85** [comment]
The project spawns the `grc_o_audio_play_stop` object, which just calls `audio_play_sound()` and `audio_stop_sound()` alternatingly on an array of audio assets, some of which has the compressed streamed attributes. When left to run for about 50 seconds by setting `alarm[0] = room_speed*50`, the `OpenAL Error` eventually shows up.

**Comment by toby-yoyo** [comment]
Thanks for the report.

We've been reworking the decoding threads (see https://github.com/YoYoGames/GameMaker-Bugs/issues/7055), which should resolve many of the underlying issues behind these errors. That rework should hopefully appear in one of the earlier 2024.1000 betas.

Encouragingly, while I can reproduce this issue on IDE v2024.8.0.169 Runtime v2024.8.0.216, I cannot on the working branch for #7055, even with the test timeout set to 10 mins. So as of now I intend to close this alongside #7055 unless something crops up before then.

**Comment by shichen85** [comment]
> Thanks for the report.
> 
> We've been reworking the decoding threads (see #7055), which should resolve many of the underlying issues behind these errors. That rework should hopefully appear in one of the earlier 2024.1000 betas.
> 
> Encouragingly, while I can reproduce this issue on IDE v2024.8.0.169 Runtime v2024.8.0.216, I cannot on the working branch for #7055, even with the test timeout set to 10 mins. So as of now I intend to close this alongside #7055 unless something crops up before then.

Hello Toby, thanks for the confirmation. We have a release target that we need to deliver before October. For the August runtime, what options do we have to work around this issue besides avoiding using the compressed streamed option?

**Comment by shichen85** [comment]
Just tried runtime 2024.1000.0.644 and still getting the errors. 
[output.txt](https://github.com/user-attachments/files/16982893/output.txt)

**Comment by toby-yoyo** [comment]
The first version with the changes from #7055 will be runtime 2024.1000.0.646.

Sorry, but I haven't been able to take a closer look at this issue just yet. I'll get back to you on the possbility of a workaround in an August version, although I would warn that it seems somewhat unlikely that there would be one.

**Comment by toby-yoyo** [comment]
@shichen85 I am now more or less free to look into the cause of this particular issue on 2024.8 if necessary (though it should not exist at all in 2024.1100). Are you still hoping to use 2024.8 for your deadline?

**Comment by toby-yoyo** [comment]
In that case, I'll close this as fixed by #7055.

**Comment by Emc1923** [comment]
Verified fixed as of  IDE v2024.1100.0.646  Runtime v2024.1100.0.666.
